library-model/library-model-generator-integration-test/src/main/resources/sample_annotated/src/com/uber/nullaway/libmodel/AnnotationExample.java

```diff
@@ -0,0 +1,34 @@
+package com.uber.nullaway.libmodel;
+
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+
+@NullMarked
+public class AnnotationExample {
+
+    @Nullable
+    public String makeUpperCase(String inputString) {
+        if (inputString == null || inputString.isEmpty()) {
+            return null;
+        } else {
+            return inputString.toUpperCase();
+        }
+    }
+
+    public Integer @Nullable [] generateIntArray(int size) {
+        if (size <= 0) {
+            return null;
+        } else {
+            Integer[] result = new Integer[size];
+            for (int i = 0; i < size; i++) {
+                result[i] = i + 1;
+            }
+            return result;
+        }
+    }
+
+    @javax.annotation.Nullable
```

Comment:
Is this to test that we _don't_ process it, since it is not from JSpecify?  If so, please document

Reply:
Wasn't there a comment here?  Did it disappear?

Reply:
That issue with the CI jobs trying to change something in the file was because of that comment for some reason, so I am trying to figure out why and have removed it till then.